airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_source.py

```diff
@@ -1,25 +0,0 @@
-#
-# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
-#
-
-from unittest.mock import MagicMock
-
-import responses
-from source_pivotal_tracker.source import SourcePivotalTracker
-
-
-@responses.activate
-def test_check_connection(config_pass, projects_response):
```

Comment:
These tests should not be deleted. Instead, they should be aligned with the new approach

Reply:
Could you please eloborate this process?
Low code cdk usually perform tests within standard acceptance-test yaml file isnt it?

Reply:
We should keep as many unit tests as possible after migration. You should use migrated source in these tests

Reply:
For this specific scenario of pivotal-tracker, the `connection check` and `streams check` are already done via standard `acceptance-test-config.yaml` file itself, these world be duplicate checks for this connector
As the gitlab connector has stream slices and oauth config for additional checks, it might require the unit tests but it would be not required here, would be my suggestion

Reply:
@lazebnyi could you please take a look at the comment above and let us know if these tests should be kept?